### PR TITLE
[12.0] [MIG] partner_address_street3

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

### CODE REVIEW

File: partner_address_street3/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    'name': 'Street3 in addresses',
+    'summary': 'Add a third address line on partners',
+    'license': 'AGPL-3',
+    'version': '12.0.1.0.0',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'maintainer': 'Camptocamp',
+    'category': 'Sales',
+    'depends': ['base'],
+    'website': 'http://www.camptocamp.com',
```
Comment (Nikul-Chaudhary) [inline partner_address_street3/__manifest__.py]: @hveficent URL should be https://github.com/OCA/partner-contact

File: partner_address_street3/tests/test_street_3.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class TestStreet3(TransactionCase):
+
+    def test_partner(self):
+        # Test address_format has been updated on existing countries
+        us_country = self.env.ref('base.us')
+        self.assertTrue('%(street3)s' in us_country.address_format)
+
+        homer = self.env['res.partner'].create({
```
Comment (Nikul-Chaudhary) [inline partner_address_street3/tests/test_street_3.py:16]: @hveficent `self.env['res.partner']` use multipal time, So use like global variable

**Review by Nikul-Chaudhary — CHANGES_REQUESTED** [review]
Improve Code

File: partner_address_street3/tests/test_street_3.py (hunk 2 above)
Comment (HviorForgeFlow) [reply partner_address_street3/tests/test_street_3.py:16]: There are only 2 matches. Avoiding nitpicking...

**Review by jcdrubay — APPROVED** [review]

**Review by tbaden — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@Nikul-Chaudhary update review

**Review by Nikul-Chaudhary — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**